helm/cluster-eks/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: cluster-eks
+home: https://github.com/giantswarm/cluster-eks
+sources:
+  - https://github.com/giantswarm/cluster-eks
+description: A helm chart for creating Cluster API EKS clusters with the AWS infrastructure provider (CAPA).
+icon: https://s.giantswarm.io/app-icons/aws/1/dark.svg
+type: application
+version: 0.1.0
+annotations:
+  application.giantswarm.io/team: "hydra"
```

Comment:
```suggestion
  application.giantswarm.io/team: "phoenix"
```

Reply:
fixed, It was actually copied from `cluster-aws` so i created a issue to fix it there as well